lint/base_linter/composer_linter.py

```diff
@@ -43,6 +43,12 @@ def context_sensitive_executable_path(self, cmd):
 
         Return a tuple of (have_path, path).
         """
+        # The default implementation will look for a user defined `executable`
+        # setting.
+        success, executable = super().context_sensitive_executable_path(cmd)
+        if success:
+            return success, executable
+
```

Comment:
This kind of duplication is not super nice but I think this is called OOP. 😉

Reply:
I don't know what the kids call it these days, but I actually like a healthy amount of duplication.